packages/core/useInfiniteScroll/index.md

```diff
@@ -35,6 +35,22 @@ useInfiniteScroll(
 </template>
 ```
 
+To stop infinite scrolling, unset the target element. This will prevent the callback from firing, and will turn off all scroll listeners.
+
+```js
+const isComplete = ref(false)
+
+useInfiniteScroll(
+  computed(() => !isComplete.value && el.value),
```

Comment:
The current example code is returning a `bool`. It should be something like this.

```suggestion
  computed(() => !isComplete.value ? el.value : null),
```

Reply:
These are essentially the same thing, the only difference is that yours returns `null` whereas mine returns `false` when it is complete. We're just doing [a truthiness check](https://github.com/vueuse/vueuse/blob/d52524441ae08c1ef7a9d3bc86c7fa32bbb0cd75/packages/core/useInfiniteScroll/index.ts#L63-L65) though, these would behave identically. They're both good options for turning off an infinite scroller 🙂

Reply:
Scott, I saw your comment in my email but appears you maybe deleted it here. I should have provided a little more context for my reasoning. I am running v10.1.2 and was having issues so I went looking and came across this example from looking at https://github.com/vueuse/vueuse/pull/3030.

I tried the example but it throws a type error because it needs the element and not a `bool`.

```
Type 'ComputedRef<false>' is not assignable to type 'Ref<Window | HTMLElement | SVGElement | Document | null | undefined>'
```

Reply:
Ya I deleted it because I was incorrect. Our could _could_ work with the original syntax, but our type signatures don't support booleans. I've committed your change! 🙂